### ISSUE TITLE
The sidebar disappear after entering a raid

### ISSUE THREAD

**Issue by Solru** [Yoctillion/Viramate-Fix#1]
The sidebar from viramate load fine when first enter game but will disappear after entering a raid. The sidebar will restore only if I quit Chrome and reopen the game. Although it will disappear again once I get into another raid.

**Comment by Yoctillion** [comment]
I don't have this problem on my computer. It may be caused by other extensions/scripts.

**Comment by Solru** [comment]
I'll test it again but I don't bave any other extension installed.

**Comment by Solru** [comment]
I tried again with a fresh chrome install with only Viramate and this extension installed and it still happen. Sometime it load fine for a few raid but it will then disappear again.

Edit: Ok I think I might have figured out what the problem is. I cannot recreate this problem if I play on the English version of the game. So I think it has something to do with that.

**Comment by Yoctillion** [comment]
I also use the Japanese version, which works fine for me.
Sometimes viramate will not work if the page has not been completely loaded. If the icon of the page is always rotating, you might need to check the network connection to the game server.

**Comment by Solru** [comment]
I don't think it has anything to do with network or stuff not loading but rather something cause the game window to offset itself. As shown here: https://puu.sh/xJCAD/1382d80090.png 
the position of the pop up would be in the right place if the side bar is actually there. 
It's not only me that has the same problem with this extension so I really don't think it's the case where my internet just sucks. 
As I mention before this bug only ever happen in the Japanese version in DMM and doesn't occur at all when you play through a mobage account in Japanese or English.

**Comment by onionknight107** [comment]
Same problem here. It not only happens after a raid, but also when I refresh the page.

**Comment by onionknight107** [comment]
Also, if I turn the game into English, it just works fine, but I really hope to play the game with Japanese.

**Comment by Yoctillion** [comment]
I have updated the script, and there is almost no difference between mobage and other versions now. If the problem still happens, it seems the cause is not the script.
